modules/startTime.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function startTime() {
+  const localTime = DateTime.now();
+  const timeNow = document.querySelector('.time');
+  timeNow.innerText = `${localTime.toLocaleString(DateTime.DATE_FULL)}, ${localTime.toLocaleString(DateTime.TIME_WITH_SECONDS)}`;
+  setTimeout(startTime, 1000);
+}
```

Comment:
- [ ] **Required:** This is a clean code and it works really good. However, it only shows the GMT time rather than the local time of the user, can we fix this?
![image](https://user-images.githubusercontent.com/27709832/148853962-fb5a4ac6-4ced-4c38-b952-b808ba5f7b36.png)

Reply:
Sure :+1: Foxed :smile: